src/main/java/com/troubleshooting/model/IAuthnUserDataModel.java

```diff
@@ -0,0 +1,13 @@
+package com.troubleshooting.model;
+
+public interface IAuthnUserDataModel {
```

Comment:
Starting an interface name with a capital I is a C# convention.
Missing documentation.

Reply:
I want to differentiate between Interfaces and Regular classes that implement them. Would `AuthenticationUserDataModelInterface` suffice?

Reply:
No not in java look at all the interfaces in Java they do not have any name identifier.
Map, Collection, List, Stream etc.